src/app/query/ContactForm.query.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { Field } from 'Util/Query';
+
+/** @namespace Query/ContactForm */
+export class ContactFormQuery {
+    getSendContactFormMutation(options) {
+        return new Field('contactForm')
+            .addArgument('contact', 'ContactForm!', options)
+            .addField('message');
```

Comment:
Please split this into separate method in order to make it more easily overridable.
With the current implementation, if custom theme's developer adds a custom field to this mutation (and to the query above - same situation), he'd need to override the query (mutation) completely, hence if afterwards we'll want to add something from our side - he will not receive this update. See the other queries in the application for a reference.

Reply:
Done.